lego/settings/celery.py

```diff
@@ -93,6 +93,10 @@ def on_setup_logging(**kwargs):
     'check-that-pool-counters-match-registration-number': {
         'task': 'lego.apps.events.tasks.check_that_pool_counters_match_registration_number',
         'schedule': crontab(hour='*', minute=0)
+    },
+    'notify_user_about_new_survey': {
+        'task': 'lego.apps.surveys.tasks.survey_mail',
+        'schedule': crontab(hour='*', minute=0)
```

Comment:
might as well ramp this up to like every tenth minute or so, it's not like it does a lot of work anyway, and that way you don't have to wait an hour if you're unlucky with the survey start time (`minute="*/10"`)